index.bs

```diff
@@ -3175,6 +3208,7 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
 1. Increment |sourceToAttribute|'s [=attribution source/number of event-level reports=] value by 1.
 1. If |matchedConfig|'s [=event-level trigger configuration/dedup key=] is not null,
     [=list/append=] it to |sourceToAttribute|'s [=attribution source/dedup keys=].
+1. [=set/Append=] |rateLimitRecord| to the [=attribution rate-limit cache=].
```

Comment:
I don't think we are supposed to add an attribution rate-limit record if `sourceToAttribute`'s randomized response is not `null`.

Reply:
Good catch, please fix,  thanks.

Reply:
To clarify -- we do insert one but elsewhere in the spec.